test/server/rest/base.spec.js

```diff
@@ -13,7 +13,8 @@ process.env.NODE_ENV = process.env.NODE_ENV || "test-rest-spec";
 
 var sinon = require("sinon");
 var app = require("../../../server");
-var PORT = process.env.TEST_REST_PORT || 3020;
+var testPort = 3020;
```

Comment:
Also, switch this one back to the way it was -- there's no readability benenfit to having another variables